### PR TITLE
Implement Basic Specifications of ToDo project

### PR DESCRIPTION
**Pull request by iliebabcenco**
In this pull request we implemented:
1. a nav section for displaying and adding new projects;
2. a main section which displays the associated todos for selected project;
3. all CRUD logic for todos;
4. creation logic for projects;
5. association between projects and todos, so a project can have many todos, and a todo belongs to one project;
6. we used localStorage for saving user's data.

### CODE REVIEW

File: dist/main.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.nav-link:hover,
+.card-link:hover {
+    cursor: pointer;
+}
+
+.card-link {
+    margin-top: 0.1rem;
+    margin-left: 0.1rem !important;
```
Comment (arslanbisharat) [inline dist/main.css]: - [x] Please don't use important in styling. This is not a good pratice.

File: dist/main.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.descr-parag {
+    color: darkblue;
+    padding-left: 1rem;
+}
+
+.nav {
+    width: 20%;
+
```
Comment (arslanbisharat) [inline dist/main.css]: - [x] Please remove unnecessary empty lines to write clean code.

File: README.md [hunk 3]
```diff
@@ -1 +1,61 @@
...
+## Live version
+
+[Todo List](https://frederickmih.github.io/TODO-LIST/)
+
+
+## How it looks
+
+![](dist/assets/mainscreen.png)
+
+## Getting Started
```
Comment (arslanbisharat) [inline README.md:18]: - [ ] [Optional] You can add more screenshots of your project in your Readme.

File: dist/main.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+.main {
+    margin: 0 auto;
+    background-color: rgb(217, 227, 236);
+    border: 1px solid gray;
+}
+
+#add-task-btn {
+    margin: 1rem auto;
+    /* padding-left: 1rem;
+    padding-right: 1rem; */
```
Comment (arslanbisharat) [inline dist/main.css]: - [x] Please remove commented code to write clean code.

File: dist/main.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.nav,
+.main {
+    margin: 0 auto;
+    background-color: rgb(217, 227, 236);
+    border: 1px solid gray;
+}
+
+#add-task-btn {
```
Comment (arslanbisharat) [inline dist/main.css]: - [x] Don't use ids as selectors. This can override other properties and make code unclean.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi Fred, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by umararshadbut — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @iliebabcenco  and @FrederickMih ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
HAPPY CODING :computer:
**Umair**

**Pull request merged**